panimg/models.py

```diff
@@ -140,6 +143,7 @@ class PanImg:
     series_instance_uid: str = ""
     study_description: str = ""
     series_description: str = ""
+    segments: Optional[Tuple[Any, ...]] = None
```

Comment:
Any reason to use `Any` rather than `np.number` here?

Reply:
Ah, I changed this after the mypy precommit check complained about it, but I had just forgotten to include the ellipsis.

Reply:
It really should only be int, floats make no sense for semantic segmentations.